fe/fe-core/src/main/java/com/starrocks/sql/ast/Identifier.java

```diff
@@ -0,0 +1,21 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.ParseNode;
+
+public class Identifier implements ParseNode {
```

Comment:
Do we really need  this class? why not use `StringLiteral`?

Reply:
Although the internal structure of StringLiteral is similar, they should not be mixed. This will lead to confusion in the code. Identifier has its own literal meaning, not what StringLiteral can represent. StringLiteral represents "Literal" in semantics, while identifier has analytic meaning.

`identifier
    : IDENTIFIER             #unquotedIdentifier
    | nonReserved            #unquotedIdentifier
    | BACKQUOTED_IDENTIFIER  #backQuotedIdentifier
    | DIGIT_IDENTIFIER       #digitIdentifier
    ;`